Pipfile

```diff
@@ -4,21 +4,22 @@ verify_ssl = true
 name = "pypi"
 
 [packages]
-boto3 = "~=1.23.0"
-logging-utilities = "~=3.0"
-Flask = "~=2.1.0"
-gevent = "~=21.12.0"
-gunicorn = "~=20.1.0"
-PyYAML = ">=5.4"
-python-dotenv = "~=0.20.0"
-validators = "~=0.19.0"
+boto3 = "~=1.28"
+logging-utilities = "~=4.0"
+Flask = "~=3.0"
```

Comment:
Did you check if they are any breaking change in version 3 ?

Reply:
As far as I can see there are no breaking changes affecting the service